nixos/jobs.nix

```
@@ -124,6 +124,17 @@
         script = ''
           set -euxo pipefail
           export SERIAL_BRIDGE_ENABLE=1
+
+          # fill the disk before running any job, or indeed before asking for
+          # any jobs to run
+
+          # need to wait for the symlink to finish up being added by udev
+          udevadm settle
+
+          # this is expected to fail once dd hits the end of the device, since
+          # we are not specifying a count. It'll fill the whole thing first, and
+          # this way it works no matter the size of it
+          dd if=/dev/urandom of=/dev/cobblestone bs=1M oflag=direct || true
```

Wait, does this exit non-zero? That's not what I would expect (and I would be worried about masking an error where we didn't finish filling the disk).